src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <article class="calendar calendar--start-day--mon calendar--month-length--31">
+      <div class="calendar__day calendar__day--1"></div>
+      <div class="calendar__day calendar__day--2"></div>
+      <div class="calendar__day calendar__day--3"></div>
```

Comment:
you can do it just with `@for` loop & nth-child
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```